src/CodeFormatter.cs

```diff
@@ -170,9 +170,6 @@ private static async Task<Workspace> OpenFolderWorkspaceAsync(string workspacePa
                 // rather than a separate one. Any tasks isolated in AppDomains or tasks that create
                 // AppDomains will likely not work due to https://github.com/Microsoft/MSBuildLocator/issues/16.
                 { "AlwaysCompileMarkupFilesInSeparateDomain", bool.FalseString },
-                // This flag is used at restore time to avoid imports from packages changing the inputs to restore,
-                // without this it is possible to get different results between the first and second restore.
-                { "ExcludeRestorePackageImports", bool.TrueString },
```

Comment:
This property was stopping NetStandard projects from resolving framework references from the project.assets.json file. A restore must be run on the project prior to formatting in order to produce this project.assets.json file, otherwise missing framework references will be reported in compiler diagnostics.